tweb/src/main/frontend/src/js/vue/components/StarredLink.vue

```diff
@@ -15,15 +13,18 @@
         @Prop() starred: boolean;
         @Prop() id: number;
 
-        public isStarred: boolean = false;
-
-        public mounted() {
-            this.isStarred = this.starred;
-        }
+		get isStarred() {
+			return {
+				fa: this.starred,
+				far: !this.starred,
+				'-starred': this.starred
+			};
+		}
 
         public async toggleStarred() {
             const request: RequestDto = await Utils.post(`/requests/${this.id}/watch`);
-            this.isStarred = request.starred;
+			this.starred = request.starred;
```

Comment:
Changing a @Prop directly it will give the error "Avoid mutating a prop directly since the value will be overwritten whenever the parent component re-renders. Instead, use a data or computed property based on the prop's value."

Therefor the variable 'isStarred' was created. Please use that. And change the 'get isStarred' to 'get starredCssClass' or something like that.